dist/assests/css/style.css

```diff
@@ -434,3 +469,43 @@ template {
 [hidden] {
   display: none;
 }
+
+/* expert trainers section */
+
+.trainers-section {
+  background-color: rgba(241, 244, 246, 255);
+  padding: 2rem;
+}
+
+.trainers-section .highlited-text {
```

Comment:
you're using BEM, you should be stating this kind of hierarchy in the CSS class name, so the class name should be something like `.trainers-section__highlighted-text`, this ensures that you keep your CSS specificity as low as possible.

Reply:
**I renamed these classes **